.changes/v3.9.0/1014-notes.md

```diff
@@ -0,0 +1,3 @@
+* Removed disk update steps from `TestAccVcdIndependentDiskBasic`, created a new one `TestAccVcdIndependentDiskBasicWithUpdates` which will be run only on new releases of VCD (< v10.4.1) [GH-1014]
```

Comment:
I don't understand. The text "new releases of VCD"  conflicts with the specification in parenthesis ("< v10.4.1"). Shouldn't be `>= v10.4.1` ?
Please elaborate

Reply:
You are correct, changed accordingly.